Backend/Backend/Services/PodcastService.cs

```diff
@@ -83,13 +86,55 @@ public class PodcastService : IPodcastService
             }
             else
             {
+
                 var podcastId = Guid.Parse(id);
                 return await _db.Podcasts!.Include(u => u.Cover).Include(u =>u.Episodes).FirstOrDefaultAsync(u => u.Id == podcastId);
         
             }
         }
 
 
+        public async Task<List<GetPodcastResponse>> GetMyPodcast(HttpContext httpContext)
+        {
+           User? user = await  _authService.IdentifyUserAsync(httpContext);
+            if(user != null)
```

Comment:
dont need to indent this. just repalace this with a guard clause. Looks somthing like this:

if (user == null) throw new Exception("Not found");
then write the rest of the code here